homeassistant/core.py

```diff
@@ -704,7 +737,7 @@ async def async_stop(self, exit_code: int = 0, *, force: bool = False) -> None:
 
         # stage 1
         self.state = CoreState.stopping
-        self.async_track_tasks()
+        self.async_create_task(self.background_tasks.async_cancel_all())
```

Comment:
This seems a bit risky to cancel all the tasks before the integration has a chance to terminate them gracefully when it gets the stop event.

Reply:
Maybe fire the stop event and than give them 10s before canceling everything?

Reply:
oh right, maybe we can move it to the 2nd stage.

Reply:
going to remove it for now.